core/src/main/java/org/mskcc/cbio/portal/dao/DaoCopyNumberSegment.java

```diff
@@ -63,6 +63,19 @@ public static int addCopyNumberSegment(CopyNumberSegment seg) throws DaoExceptio
             return 1;
         }
     }
+
+    public static int addFractionGenomeAltered(int cancerStudyId, int sampleId) throws DaoException {
```

Comment:
Instead of doing it one sample at a time maybe we can do it for the whole study (and later for the whole copy number segment profile once we have that). similar to what we did for mutation_count: https://github.com/cBioPortal/cbioportal/blob/master/core/src/main/java/org/mskcc/cbio/portal/dao/DaoMutation.java#L133

Reply:
Once I convert it to a SQL-only solution, I think I can do it with just one insert query per study.